conda-arm64-requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Required for v3io-frames client through grpcio-tools:
 # https://github.com/v3io/frames/blob/5490b1e7f281a8c44ec679105957e181c3fa81fc/clients/py/requirements.txt#L4
 # grpcio-tools 1.48.2 requires protobuf<4.0dev,>=3.12.0
-protobuf>=3.20.3, <4
+protobuf>=3.20.3,<4
```

Comment:
So how does this work now without the python version conditional?

Reply:
well, as my comment above stated, Im leaving it for now until python3.11 is fully supported for development-automation flow